### PR TITLE
Editable Reblogs: make this work for the HTML text editor

### PR DESCRIPTION
**Pull request by bvtsang**
For issue #64.

Thanks to @hobinjk for helping out!

### CODE REVIEW

File: Extensions/xkit_patches.js [hunk 1]
```diff
@@ -967,15 +967,37 @@ XKit.tools.get_blogs = function() {
...
+						XKit.tools.add_function(function(new_content){
+							var editor_div = document.getElementsByClassName("ace_editor");
+							if (editor_div.length === 1) {
+								var editor = window.ace.edit(editor_div[0]);
+								editor.setValue(new_content);
+								setTimeout(function(){
+									jQuery(".ace_marker-layer").empty();
+								}, 500);
+							}
+						}, true, "", new_content);
```
Comment (hobinjk) [inline Extensions/xkit_patches.js]: The changes to xkit.js are unnecessary if you use the `addt` argument of `add_function`, replacing the "" with JSON.stringify(new_content) and beginning the function with `var new_content = add_tag`.

Comment (bvtsang) [inline Extensions/xkit_patches.js]: Makes sense! I was a bit afraid to modify `add_function`, so you proved my fears correct. Thanks for the review.

**Comment by bvtsang** [comment]
I didn't wrap `new_content` with `JSON.stringify` in my next commit because that would have resulted in wrapping the reblogged text in extra quotation marks.

**Comment by hobinjk** [comment]
Great! Thanks for making the change! I'll merge and push this now

**Pull request merged**